docs/UserGuide.md

```diff
@@ -82,49 +96,48 @@ Some examples of **invalid** user inputs for the *add command* are provided belo
 
 ## Features
 
-
-
 ### Viewing help : `help`
 
-Provides a link to the full User Guide that users can visit by clicking on the button 'Open in  Browser'.
-
-![help message](images/helpMessage.png)
+Provides a reference for commands in StudyTracker.
+You can also visit the User Guide and Developer Guide by clicking on the buttons in the window.
+![Help window](images/helpMessage.png)
 
 Format: `help`
 
-
 ### Adding a study spot: `add`
 
 Adds a study spot to the StudyTracker.
 
-Format: `add n/NAME* r/RATING* a/ADDRESS t/TAG... m/AMENITY...`
+**Format:** `add n/NAME* r/RATING* a/ADDRESS t/TAG... m/AMENITY...`
 
 <div markdown="span" class="alert alert-primary">:bulb: **Tip:**
 Tags, amenities and address are all optional.
 </div>
 
-Examples:
+**Examples:**
 ```
 add n/COM1 Basement r/5
 New study spot “COM1 Basement” added to list!
 
-add n/Starbucks at U-Town r/4 t/noisy m/wifi
-New study spot “Starbucks at U-Town” added to list!
+add n/Starbucks at UTown r/4 t/noisy m/wifi
+New study spot “Starbucks at UTown” added to list!
 ```
 
 ### Listing study spots : `list`
 
 Shows saved study spots in the StudyTracker.
 
-Format: `list` to show all study spots
+**Format:** `list` to show all study spots
 
-Format: `list -f` to show all favourite study spots
+**Format:** `list -f` to show all favourite study spots
 
-Format: `list -t t/TAG...` to show study spots with specified tags
+**Format:** `list -t t/TAG...` to show study spots with specified tags
 
-* Multiple flags can be used at once
+<div markdown="span" class="alert alert-primary">:bulb: **Tip:**
+Multiple flags can be used at once.
+</div>
 
-Examples:
+**Examples:**
 ```
 list 
```

Comment:
Not sure how we can display the expected output when it's mostly GUI

E.g. 
when we enter the command `list -t t/noisy`, the actual output to users is `Listed all study spots with Tags: [noisy]`

when we enter the command `list`, actual output: `Listed all study spots`

when we enter the command `list -f`, actual output: `Listed all study spots in Favourites`